README.md

```diff
@@ -1,5 +1,20 @@
 # Infrahub documentation website
 
+⚠️ **Important Notice**: This repository is a **documentation aggregation site** that combines documentation from multiple Infrahub repositories. Most of the documentation content here is **read-only** and automatically synced from other repositories.
+
+## Before opening an issue or pull request
+
+**Please do not open issues or pull requests for documentation content in this repository.** Instead, please open them in the appropriate source repository where the documentation actually lives:
+
+- **Infrahub core documentation**: [opsmill/infrahub](https://github.com/opsmill/infrahub) repository
+- **Python SDK documentation**: [opsmill/infrahub-sdk-python](https://github.com/opsmill/infrahub-sdk-python) repository  
+- **Ansible collection documentation**: [opsmill/infrahub-ansible](https://github.com/opsmill/infrahub-ansible) repository
+- **Other integrations**: Check the specific integration repository in the [OpsMill GitHub organization](https://github.com/opsmill)
+
```

Comment:
Can you add the full list of repository here 

it's missing the following one 
- **Service catalog documentation**: [opsmill/infrahub-demo-service-catalog](https://github.com/opsmill/infrahub-demo-service-catalog) repository
- **Nornir integration documentation**: [opsmill/nornir-infrahub](https://github.com/opsmill/nornir-infrahub) repository
- **Schema library documentation**: [opsmill/schema-library](https://github.com/opsmill/schema-library) repository
- **Infrahub Sync documentation**: [opsmill/infrahub-sync](https://github.com/opsmill/infrahub-sync) repository
- **Emma documentation**: [opsmill/emma](https://github.com/opsmill/emma) repository